libs/sdk-ui-kit/src/@ui/UiFocusTrap/UiFocusTrap.tsx

```diff
@@ -146,6 +165,7 @@ export const UiFocusTrap: React.FC<UiFocusTrapProps> = ({
     initialFocus,
     refocusKey,
     customKeyboardNavigationHandler,
+    isTabCaught = true,
```

Comment:
UiFocusTrap should be split in next iteration to do the FocusTrap part only and managing other focus related tasks should be moved to some new component used directly on places where trapping is not needed and internally by FocusTrap